Leverage std::numbers more for some of our constants

cdumez opened the pull request:
#### 91bbe4c98b2aff226120fb71c98b32b62130f157
<pre>
Leverage std::numbers more for some of our constants
<a href="https://bugs.webkit.org/show_bug.cgi?id=292372">https://bugs.webkit.org/show_bug.cgi?id=292372</a>

Reviewed by Sam Weinig.

This avoids duplicating them in wtf/MathExtras.h.

* Source/JavaScriptCore/b3/testb3.h:
(populateWithInterestingValues):
* Source/JavaScriptCore/b3/testb3_3.cpp:
(testDoubleToFloatThroughPhi):
(addArgTests):
* Source/JavaScriptCore/b3/testb3_6.cpp:
(testSelectDoubleCompareFloat):
* Source/JavaScriptCore/runtime/MathObject.cpp:
(JSC::MathObject::finishCreation):
* Source/WTF/wtf/MathExtras.h:
* Source/WebCore/Modules/webaudio/PannerNode.cpp:
(WebCore::PannerNode::calculateAzimuthElevation):
* Source/WebCore/Modules/webaudio/PeriodicWave.cpp:
(WebCore::PeriodicWave::generateBasicWaveform):
* Source/WebCore/Modules/webaudio/RealtimeAnalyser.cpp:
* Source/WebCore/Modules/webxr/WebXRSession.h:
* Source/WebCore/css/calc/CSSCalcTree+Parser.cpp:
(WebCore::CSSCalc::lookupConstantNumber):
* Source/WebCore/dom/PointerEvent.cpp:
(WebCore::PointerEvent::angleFromTilt):
* Source/WebCore/html/canvas/CanvasPath.cpp:
(WebCore::normalizeAngles):
* Source/WebCore/page/cocoa/ResourceUsageOverlayCocoa.mm:
* Source/WebCore/platform/DictationCaretAnimator.cpp:
(WebCore::keyframe):
* Source/WebCore/platform/audio/Biquad.cpp:
(WebCore::Biquad::setLowpassParams):
(WebCore::Biquad::setHighpassParams):
(WebCore::Biquad::setLowShelfParams):
(WebCore::Biquad::setHighShelfParams):
(WebCore::Biquad::setPeakingParams):
(WebCore::Biquad::setAllpassParams):
(WebCore::Biquad::setNotchParams):
(WebCore::Biquad::setBandpassParams):
(WebCore::Biquad::getFrequencyResponse):
* Source/WebCore/platform/audio/DownSampler.cpp:
(WebCore::DownSampler::initializeKernel):
* Source/WebCore/platform/audio/DynamicsCompressorKernel.cpp:
(WebCore::DynamicsCompressorKernel::process):
* Source/WebCore/platform/audio/FFTFrame.cpp:
(WebCore::FFTFrame::interpolateFrequencyComponents):
(WebCore::FFTFrame::extractAverageGroupDelay):
(WebCore::FFTFrame::addConstantGroupDelay):
* Source/WebCore/platform/audio/IIRFilter.cpp:
(WebCore::IIRFilter::getFrequencyResponse):
* Source/WebCore/platform/audio/SincResampler.cpp:
(WebCore::SincResampler::initializeKernel):
* Source/WebCore/platform/audio/UpSampler.cpp:
(WebCore::UpSampler::initializeKernel):
* Source/WebCore/platform/calc/CalculationExecutor.h:
(WebCore::Calculation::OperatorExecutor&lt;Tan&gt;::operator()):
* Source/WebCore/platform/graphics/FloatRoundedRect.cpp:
(WebCore::approximateAsRegion):
* Source/WebCore/platform/graphics/GeometryUtilities.cpp:
(WebCore::rotatedBoundingRectWithMinimumAngleOfRotation):
* Source/WebCore/platform/graphics/PathSegmentData.cpp:
(WebCore::angleOfLine):
(WebCore::calculateArcToEndPoint):
(WebCore::PathArc::extendBoundingRect const):
* Source/WebCore/platform/graphics/PathUtilities.cpp:
(WebCore::walkGraphAndExtractPolygon):
* Source/WebCore/platform/graphics/ShadowBlur.cpp:
(WebCore::calculateLobes):
* Source/WebCore/platform/graphics/avfoundation/objc/LocalSampleBufferDisplayLayer.mm:
(WebCore::transformationMatrixForVideoFrame):
* Source/WebCore/platform/graphics/avfoundation/objc/MediaPlayerPrivateMediaStreamAVFObjC.mm:
(WebCore::videoTransformationMatrix):
* Source/WebCore/platform/graphics/cairo/CairoOperations.cpp:
(WebCore::Cairo::drawEllipse):
* Source/WebCore/platform/graphics/cairo/GradientCairo.cpp:
(WebCore::addConicSector):
* Source/WebCore/platform/graphics/cairo/PathCairo.cpp:
(WebCore::PathCairo::add):
* Source/WebCore/platform/graphics/displaylists/DisplayListRecorder.cpp:
(WebCore::DisplayList::Recorder::updateStateForRotate):
* Source/WebCore/platform/graphics/filters/FEGaussianBlur.cpp:
(WebCore::gaussianKernelFactor):
* Source/WebCore/platform/graphics/skia/PathSkia.cpp:
(WebCore::PathSkia::addEllipse):
* Source/WebCore/platform/graphics/transforms/AffineTransform.cpp:
(WebCore::AffineTransform::blend):
* Source/WebCore/platform/ios/WebCoreMotionManager.mm:
(-[WebCoreMotionManager sendMotionData:withHeading:]):
* Source/WebCore/platform/mediarecorder/MediaRecorderPrivateEncoder.cpp:
(WebCore::MediaRecorderPrivateEncoder::appendVideoFrame):
* Source/WebCore/platform/mediastream/gstreamer/MockRealtimeAudioSourceGStreamer.cpp:
* Source/WebCore/platform/mediastream/mac/MockAudioSharedUnit.mm:
* Source/WebCore/platform/mock/MockRealtimeVideoSource.cpp:
(WebCore::MockRealtimeVideoSource::drawAnimation):
* Source/WebCore/rendering/ios/RenderThemeIOS.mm:
(WebCore::paintAttachmentProgress):
* Source/WebCore/rendering/mathml/MathOperator.cpp:
(WebCore::MathOperator::calculateDisplayStyleLargeOperator):
* Source/WebCore/rendering/mathml/RenderMathMLMenclose.cpp:
(WebCore::RenderMathMLMenclose::spaceAroundContent const):
(WebCore::RenderMathMLMenclose::paint):
* Source/WebCore/rendering/svg/SVGLayerTransformComputation.h:
(WebCore::SVGLayerTransformComputation::calculateScreenFontSizeScalingFactor const):
* Source/WebCore/rendering/svg/SVGRenderSupport.cpp:
(WebCore::SVGRenderSupport::calculateApproximateStrokeBoundingBox):
* Source/WebCore/rendering/svg/SVGRenderingContext.cpp:
(WebCore::SVGRenderingContext::calculateScreenFontSizeScalingFactor):
* Source/WebCore/style/values/shapes/StyleCircleFunction.cpp:
(WebCore::Style::resolveRadius):
* Source/WebCore/svg/SVGLengthContext.cpp:
(WebCore::dimensionForLengthMode):
* Source/WebCore/svg/SVGPathParser.cpp:
(WebCore::SVGPathParser::decomposeArcToCubic):
* Source/WebCore/svg/graphics/filters/SVGFilter.cpp:
(WebCore::SVGFilter::resolvedPoint3D const):
* Source/WebKit/UIProcess/API/Cocoa/_WKTouchEventGenerator.mm:
(simpleCurveInterpolation):
* Source/WebKit/UIProcess/ios/WKKeyboardScrollingAnimator.mm:
(-[WKKeyboardScrollingAnimator beginWithEvent:]):
* Source/WebKit/UIProcess/ios/forms/WKFormSelectPicker.mm:
(-[WKSelectPickerGroupHeaderView setCollapsed:animated:]):
* Source/WebKitLegacy/ios/WebView/WebPDFViewIOS.mm:
(-[WebPDFView _computePageRects]):
* Source/WebKitLegacy/ios/WebView/WebPDFViewPlaceholder.mm:
(-[WebPDFViewPlaceholder _getPDFPageBounds:]):
* Source/WebKitLegacy/mac/WebView/WebWindowAnimation.mm:
(-[WebWindowScaleAnimation currentValue]):
* Tools/TestWebKitAPI/Tests/WTF/AtomString.cpp:
(TestWebKitAPI::TEST(WTF, AtomStringNumberDouble)):
* Tools/TestWebKitAPI/Tests/WTF/MediaTime.cpp:
(TestWebKitAPI::TEST(WTF, MediaTime)):
* Tools/TestWebKitAPI/Tests/WTF/WTFString.cpp:
(TestWebKitAPI::TEST(WTF, StringNumberFixedPrecision)):
(TestWebKitAPI::TEST(WTF, StringNumberFixedWidth)):
(TestWebKitAPI::TEST(WTF, StringNumber)):
* Tools/TestWebKitAPI/Tests/WebCore/DisplayListRecorderTests.cpp:
* Tools/TestWebKitAPI/Tests/WebCore/cg/BifurcatedGraphicsContextTestsCG.cpp:
(TestWebKitAPI::TEST(BifurcatedGraphicsContextTests, TransformedClip)):
* Tools/WebKitTestRunner/ios/HIDEventGenerator.mm:
(simpleCurveInterpolation):
(-[HIDEventGenerator stylusDownAtPoint:azimuthAngle:altitudeAngle:pressure:]):

Canonical link: <a href="https://commits.webkit.org/294398@main">https://commits.webkit.org/294398@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/2503903b99226532db46238eeb9a3fa802b413da

| Misc | iOS, visionOS, tvOS & watchOS  | macOS  | Linux |  Windows |
| ----- | ---------------------- | ------- |  ----- |  --------- |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/101669 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/131/builds/21337 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/138/builds/11652 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/106827 "Built successfully") | [✅ 🛠 win](https://ews-build.webkit.org/#/builders/59/builds/52303 "Built successfully") 
| [✅ 🧪 bindings](https://ews-build.webkit.org/#/builders/9/builds/103709 "Passed tests") | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/130/builds/21645 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/123/builds/29837 "Built successfully") | [  ~~🧪 wpe-wk2~~](https://ews-build.webkit.org/#/builders/34/builds/77425 "The change is no longer eligible for processing. Commit was outdated when EWS attempted to process it.") | [✅ 🧪 win-tests](https://ews-build.webkit.org/#/builders/60/builds/34453 "Passed tests") 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/104676 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/132/builds/16726 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/91814 "Passed tests") | [✅ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/57762 "Passed tests") | 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/133/builds/16552 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/135/builds/9831 "Passed tests") | [✅ 🛠 wpe-cairo](https://ews-build.webkit.org/#/builders/65/builds/51652 "Built successfully") | 
| [✅ 🛠 🧪 jsc](https://ews-build.webkit.org/#/builders/20/builds/94341 "Built successfully and passed tests") | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/86418 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/136/builds/9908 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/109181 "Built successfully") | 
| [✅ 🛠 🧪 jsc-arm64](https://ews-build.webkit.org/#/builders/12/builds/100279 "Built successfully and passed tests") | [✅ 🛠 vision](https://ews-build.webkit.org/#/builders/128/builds/28802 "Built successfully") | [  ~~🧪 mac-AS-debug-wk2~~](https://ews-build.webkit.org/#/builders/122/builds/21211 "The change is no longer eligible for processing. Commit was outdated when EWS attempted to process it.") | [  ~~🧪 gtk-wk2~~](https://ews-build.webkit.org/#/builders/1/builds/86402 "The change is no longer eligible for processing. Commit was outdated when EWS attempted to process it.") | 
| | [✅ 🛠 vision-sim](https://ews-build.webkit.org/#/builders/121/builds/29163 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/88015 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/85967 "Passed tests") | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/19/builds/21878 "Built successfully and passed tests") | [✅ 🧪 vision-wk2](https://ews-build.webkit.org/#/builders/126/builds/30718 "Passed tests") | [✅ 🧪 mac-intel-wk2](https://ews-build.webkit.org/#/builders/137/builds/8432 "Passed tests") | [✅ 🛠 playstation](https://ews-build.webkit.org/#/builders/134/builds/22965 "Built successfully") | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/127/builds/28730 "Built successfully") | [✅ 🛠 mac-safer-cpp](https://ews-build.webkit.org/#/builders/120/builds/34019 "Built successfully") | [✅ 🛠 jsc-armv7](https://ews-build.webkit.org/#/builders/35/builds/123903 "Built successfully") | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/125/builds/28541 "Built successfully") | | [✅ 🧪 jsc-armv7-tests](https://ews-build.webkit.org/#/builders/25/builds/34428 "Passed tests") | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/129/builds/31864 "Built successfully") | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/124/builds/30100 "Built successfully") | | | 
<!--EWS-Status-Bubble-End-->

weinig commented on Source/WTF/wtf/MathExtras.h:
```diff
@@ -108,13 +75,13 @@ extern "C" inline double wtf_atan2(double x, double y)
 constexpr double turnsPerDegreeDouble = 1.0 / 360.0;
 constexpr double degreesPerTurnDouble = 360.0;
-constexpr double radiansPerTurnDouble = 2.0f * piDouble;
+constexpr double radiansPerTurnDouble = 2.0f * std::numbers::pi;
```
This is an existing bug, but that the `2.0f` should just be `2.0`.

weinig commented on Source/WebCore/Modules/webxr/WebXRSession.h:
```diff
@@ -167,7 +168,7 @@ class WebXRSession final : public RefCounted<WebXRSession>, public EventTarget,
 
     double m_minimumInlineFOV { 0.0 };
-    double m_maximumInlineFOV { piFloat };
+    double m_maximumInlineFOV { std::numbers::pi_v<float> };
```
This is also existing, but using the float version for this double value seems incorrect.

weinig commented on Source/WebCore/platform/DictationCaretAnimator.cpp:
```diff
@@ -46,7 +47,7 @@ static constexpr KeyFrame keyframe(size_t i)
     i %= updateRate;
     constexpr float inverseFrameRate = 1.f / static_cast<float>(updateRate);
-    return KeyFrame { Seconds(i * inverseFrameRate), fabs(sinf(static_cast<float>(M_PI * i * inverseFrameRate))) };
+    return KeyFrame { Seconds(i * inverseFrameRate), fabs(sinf(static_cast<float>(std::numbers::pi * i * inverseFrameRate))) };
```
oof, the choices of floats vs. doubles here seems suspect.

weinig commented on Source/WebCore/rendering/ios/RenderThemeIOS.mm:
```diff
@@ -1312,7 +1313,7 @@ static void paintAttachmentProgress(GraphicsContext& context, AttachmentLayout&
     progressPath.moveTo(center);
     progressPath.addLineTo(FloatPoint(center.x(), info.progressRect.y()));
-    progressPath.addArc(center, info.progressRect.width() / 2, -M_PI_2, info.progress * 2 * M_PI - M_PI_2, RotationDirection::Counterclockwise);
+    progressPath.addArc(center, info.progressRect.width() / 2, -M_PI_2, info.progress * 2 * std::numbers::pi - M_PI_2, RotationDirection::Counterclockwise);
```
Can we get rid of the M_PI_2 here as well?

cdumez replied:
I can use `piOverTwoDouble` instead, there is no corresponding constant in std::numbers. Is this what you had in mind?

weinig replied:
Yeah. Just seemed weird to mix.

cdumez replied:
Ok. The last iteration took care of this. Thanks.

cdumez committed:
Leverage std::numbers more for some of our constants
https://bugs.webkit.org/show_bug.cgi?id=292372

Reviewed by Sam Weinig.

This avoids duplicating them in wtf/MathExtras.h.

* Source/JavaScriptCore/b3/testb3.h:
(populateWithInterestingValues):
* Source/JavaScriptCore/b3/testb3_3.cpp:
(testDoubleToFloatThroughPhi):
(addArgTests):
* Source/JavaScriptCore/b3/testb3_6.cpp:
(testSelectDoubleCompareFloat):
* Source/JavaScriptCore/runtime/MathObject.cpp:
(JSC::MathObject::finishCreation):
* Source/WTF/wtf/MathExtras.h:
* Source/WebCore/Modules/webaudio/PannerNode.cpp:
(WebCore::PannerNode::calculateAzimuthElevation):
* Source/WebCore/Modules/webaudio/PeriodicWave.cpp:
(WebCore::PeriodicWave::generateBasicWaveform):
* Source/WebCore/Modules/webaudio/RealtimeAnalyser.cpp:
* Source/WebCore/Modules/webxr/WebXRSession.h:
* Source/WebCore/css/calc/CSSCalcTree+Parser.cpp:
(WebCore::CSSCalc::lookupConstantNumber):
* Source/WebCore/dom/PointerEvent.cpp:
(WebCore::PointerEvent::angleFromTilt):
* Source/WebCore/html/canvas/CanvasPath.cpp:
(WebCore::normalizeAngles):
* Source/WebCore/page/cocoa/ResourceUsageOverlayCocoa.mm:
* Source/WebCore/platform/DictationCaretAnimator.cpp:
(WebCore::keyframe):
* Source/WebCore/platform/audio/Biquad.cpp:
(WebCore::Biquad::setLowpassParams):
(WebCore::Biquad::setHighpassParams):
(WebCore::Biquad::setLowShelfParams):
(WebCore::Biquad::setHighShelfParams):
(WebCore::Biquad::setPeakingParams):
(WebCore::Biquad::setAllpassParams):
(WebCore::Biquad::setNotchParams):
(WebCore::Biquad::setBandpassParams):
(WebCore::Biquad::getFrequencyResponse):
* Source/WebCore/platform/audio/DownSampler.cpp:
(WebCore::DownSampler::initializeKernel):
* Source/WebCore/platform/audio/DynamicsCompressorKernel.cpp:
(WebCore::DynamicsCompressorKernel::process):
* Source/WebCore/platform/audio/FFTFrame.cpp:
(WebCore::FFTFrame::interpolateFrequencyComponents):
(WebCore::FFTFrame::extractAverageGroupDelay):
(WebCore::FFTFrame::addConstantGroupDelay):
* Source/WebCore/platform/audio/IIRFilter.cpp:
(WebCore::IIRFilter::getFrequencyResponse):
* Source/WebCore/platform/audio/SincResampler.cpp:
(WebCore::SincResampler::initializeKernel):
* Source/WebCore/platform/audio/UpSampler.cpp:
(WebCore::UpSampler::initializeKernel):
* Source/WebCore/platform/calc/CalculationExecutor.h:
(WebCore::Calculation::OperatorExecutor<Tan>::operator()):
* Source/WebCore/platform/graphics/FloatRoundedRect.cpp:
(WebCore::approximateAsRegion):
* Source/WebCore/platform/graphics/GeometryUtilities.cpp:
(WebCore::rotatedBoundingRectWithMinimumAngleOfRotation):
* Source/WebCore/platform/graphics/PathSegmentData.cpp:
(WebCore::angleOfLine):
(WebCore::calculateArcToEndPoint):
(WebCore::PathArc::extendBoundingRect const):
* Source/WebCore/platform/graphics/PathUtilities.cpp:
(WebCore::walkGraphAndExtractPolygon):
* Source/WebCore/platform/graphics/ShadowBlur.cpp:
(WebCore::calculateLobes):
* Source/WebCore/platform/graphics/avfoundation/objc/LocalSampleBufferDisplayLayer.mm:
(WebCore::transformationMatrixForVideoFrame):
* Source/WebCore/platform/graphics/avfoundation/objc/MediaPlayerPrivateMediaStreamAVFObjC.mm:
(WebCore::videoTransformationMatrix):
* Source/WebCore/platform/graphics/cairo/CairoOperations.cpp:
(WebCore::Cairo::drawEllipse):
* Source/WebCore/platform/graphics/cairo/GradientCairo.cpp:
(WebCore::addConicSector):
* Source/WebCore/platform/graphics/cairo/PathCairo.cpp:
(WebCore::PathCairo::add):
* Source/WebCore/platform/graphics/displaylists/DisplayListRecorder.cpp:
(WebCore::DisplayList::Recorder::updateStateForRotate):
* Source/WebCore/platform/graphics/filters/FEGaussianBlur.cpp:
(WebCore::gaussianKernelFactor):
* Source/WebCore/platform/graphics/skia/PathSkia.cpp:
(WebCore::PathSkia::addEllipse):
* Source/WebCore/platform/graphics/transforms/AffineTransform.cpp:
(WebCore::AffineTransform::blend):
* Source/WebCore/platform/ios/WebCoreMotionManager.mm:
(-[WebCoreMotionManager sendMotionData:withHeading:]):
* Source/WebCore/platform/mediarecorder/MediaRecorderPrivateEncoder.cpp:
(WebCore::MediaRecorderPrivateEncoder::appendVideoFrame):
* Source/WebCore/platform/mediastream/gstreamer/MockRealtimeAudioSourceGStreamer.cpp:
* Source/WebCore/platform/mediastream/mac/MockAudioSharedUnit.mm:
* Source/WebCore/platform/mock/MockRealtimeVideoSource.cpp:
(WebCore::MockRealtimeVideoSource::drawAnimation):
* Source/WebCore/rendering/ios/RenderThemeIOS.mm:
(WebCore::paintAttachmentProgress):
* Source/WebCore/rendering/mathml/MathOperator.cpp:
(WebCore::MathOperator::calculateDisplayStyleLargeOperator):
* Source/WebCore/rendering/mathml/RenderMathMLMenclose.cpp:
(WebCore::RenderMathMLMenclose::spaceAroundContent const):
(WebCore::RenderMathMLMenclose::paint):
* Source/WebCore/rendering/svg/SVGLayerTransformComputation.h:
(WebCore::SVGLayerTransformComputation::calculateScreenFontSizeScalingFactor const):
* Source/WebCore/rendering/svg/SVGRenderSupport.cpp:
(WebCore::SVGRenderSupport::calculateApproximateStrokeBoundingBox):
* Source/WebCore/rendering/svg/SVGRenderingContext.cpp:
(WebCore::SVGRenderingContext::calculateScreenFontSizeScalingFactor):
* Source/WebCore/style/values/shapes/StyleCircleFunction.cpp:
(WebCore::Style::resolveRadius):
* Source/WebCore/svg/SVGLengthContext.cpp:
(WebCore::dimensionForLengthMode):
* Source/WebCore/svg/SVGPathParser.cpp:
(WebCore::SVGPathParser::decomposeArcToCubic):
* Source/WebCore/svg/graphics/filters/SVGFilter.cpp:
(WebCore::SVGFilter::resolvedPoint3D const):
* Source/WebKit/UIProcess/API/Cocoa/_WKTouchEventGenerator.mm:
(simpleCurveInterpolation):
* Source/WebKit/UIProcess/ios/WKKeyboardScrollingAnimator.mm:
(-[WKKeyboardScrollingAnimator beginWithEvent:]):
* Source/WebKit/UIProcess/ios/forms/WKFormSelectPicker.mm:
(-[WKSelectPickerGroupHeaderView setCollapsed:animated:]):
* Source/WebKitLegacy/ios/WebView/WebPDFViewIOS.mm:
(-[WebPDFView _computePageRects]):
* Source/WebKitLegacy/ios/WebView/WebPDFViewPlaceholder.mm:
(-[WebPDFViewPlaceholder _getPDFPageBounds:]):
* Source/WebKitLegacy/mac/WebView/WebWindowAnimation.mm:
(-[WebWindowScaleAnimation currentValue]):
* Tools/TestWebKitAPI/Tests/WTF/AtomString.cpp:
(TestWebKitAPI::TEST(WTF, AtomStringNumberDouble)):
* Tools/TestWebKitAPI/Tests/WTF/MediaTime.cpp:
(TestWebKitAPI::TEST(WTF, MediaTime)):
* Tools/TestWebKitAPI/Tests/WTF/WTFString.cpp:
(TestWebKitAPI::TEST(WTF, StringNumberFixedPrecision)):
(TestWebKitAPI::TEST(WTF, StringNumberFixedWidth)):
(TestWebKitAPI::TEST(WTF, StringNumber)):
* Tools/TestWebKitAPI/Tests/WebCore/DisplayListRecorderTests.cpp:
* Tools/TestWebKitAPI/Tests/WebCore/cg/BifurcatedGraphicsContextTestsCG.cpp:
(TestWebKitAPI::TEST(BifurcatedGraphicsContextTests, TransformedClip)):
* Tools/WebKitTestRunner/ios/HIDEventGenerator.mm:
(simpleCurveInterpolation):
(-[HIDEventGenerator stylusDownAtPoint:azimuthAngle:altitudeAngle:pressure:]):

Canonical link: https://commits.webkit.org/294398@main
